doc/README/Data/Tree/AVL.agda

```diff
@@ -28,6 +28,21 @@ open import Relation.Binary.PropositionalEquality
 open Data.Tree.AVL <-strictTotalOrder renaming (Tree to Tree′)
 Tree = Tree′ (MkValue (Vec String) (subst (Vec String)))
 
+-- The first argument to MkValue should be a function from a key
+-- (a ℕ in this case) to a value type (a String vector type).
+-- [Since (Vec String) is missing Vec's second parameter, a ℕ,
```

Comment:
Code is usually wrapped in backticks (e.g. `Vec String`) rather than brackets by convention. It would be good if you could wrap other code in them as well, e.g. `MkValue`